inc/dbmysqliterator.class.php

```diff
@@ -468,11 +468,18 @@ public function analyseCrit ($crit, $bool = "AND") {
                trigger_error("BAD FOREIGN KEY, should be [ key1, key2 ]", E_USER_ERROR);
             }
 
+         } else if ($name === 'RAW') {
+            $key = key($value);
+            $value = current($value);
+            $ret .= '((' . $key . ') = ' . $this->analyseCrit($value) . ')';
          } else if (is_array($value)) {
             if (count($value) == 2
                   && isset($value[0]) && in_array($value[0], $operators, true)) {
-
-               $ret .= DBmysql::quoteName($name) . " {$value[0]} " . DBmysql::quoteValue($value[1]);
+               if ($value[0] === '|=') {
```

Comment:
I think it could be done using the `RAW` keyword, in order to avoid adding this kind of complexity just for a specific case.